Home_work14/Home_work14.cpp

```diff
@@ -1,32 +1,42 @@
 ﻿#include <iostream>
 #include <string>
+#include <vector>
+#include <algorithm>
+
+class Veapon; class Player; class Team;
 enum class Role {
 	Assault,
 	Support,
 	Sniper,
 	Medic
 };
 class Rectangle {
+private:
+	float lenght{}, hight{};
 public:
-	float leght{}, hight{};
-	float getArea() {
-		return leght * hight;
+	
+	const float getArea() {
```

Comment:
float getArea() const {
}
with const float getArea() 
you make the return value const, not the method